jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpan.java

```diff
@@ -75,6 +75,10 @@ public long getStart() {
     return startTimeMicroseconds;
   }
 
+  public boolean isFinished() {
+    return finished;
```

Comment:
needs `synchronized (this)`, see finish() method

Reply:
updated